PowerShell.Common.props

```diff
@@ -93,9 +95,8 @@
     <Company>Microsoft Corporation</Company>
     <Copyright>(c) Microsoft Corporation. All rights reserved.</Copyright>
 
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <RuntimeFrameworkVersion>2.1.8</RuntimeFrameworkVersion>
-    <LangVersion>Latest</LangVersion>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <LangVersion>8.0</LangVersion>
```

Comment:
I think this can be changed back to latest. Projects targetting netcoreapp3.0 default to langversion 8.0. https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version

Reply:
`latest` means latest stable version.  8.0 is in preview.  Prefer explicit 8.0 to preview which is ambiguous.